static/styles/left_sidebar.css

```diff
@@ -195,8 +195,7 @@ li.show-more-topics {
 
 #add-stream-link {
     text-decoration: none;
-    margin-left: 10px;
-    margin-top: 5px;
+    margin: 5px auto 5.5px 10px;
```

Comment:
I feel like long term we're going to want to make the CSS more obviously identical for the bottom alignment of the bottom components in the sidebars, but I don't think that's a blocker for merging this pull request, which definitely makes things better.